gammapy/makers/safe.py

```diff
@@ -170,6 +170,12 @@ def make_mask_energy_aeff_max(self, dataset, observation=None):
             position = geom.center_skydir
 
         aeff = exposure.get_spectrum(position) / exposure.meta["livetime"]
+        if not np.any(aeff.data > 0.):
+            log.warning(
+                f"No safe energy band can be defined for the dataset {dataset.name}: setting mask_safe to False"
```

Comment:
Maybe dd the info on the method here:
```suggestion
                f"Effective area is all zero at {position}. No safe energy band can be defined for the dataset '{dataset.name}': setting `mask_safe` to all False."
```